lib/Ed25519VerificationKey2020.js

```diff
@@ -31,10 +39,24 @@ export class Ed25519VerificationKey2020 extends LDKeyPair {
     super(options);
     this.type = SUITE_ID;
     this.publicKeyMultibase = options.publicKeyMultibase;
+    this.privateKeyMultibase = options.privateKeyMultibase;
+
     if(!this.publicKeyMultibase) {
       throw new TypeError('The "publicKeyMultibase" property is required.');
     }
-    this.privateKeyMultibase = options.privateKeyMultibase;
+
+    if(!this.publicKeyMultibase || !_isValidKeyHeader(
+      this.publicKeyMultibase, MULTICODEC_ED25519_PUB_HEADER)) {
+      throw new Error('`publicKeyMultibase` has invalid byte header: ' +
+      `"${this.publicKeyMultibase}".`);
+    }
+
+    if(this.privateKeyMultibase && !_isValidKeyHeader(
+      this.privateKeyMultibase, MULTICODEC_ED25519_PRIV_HEADER)) {
+      throw new Error('`privateKeyMultibase` has invalid byte header: ' +
+       `"${this.privateKeyMultibase}".`);
```

Comment:
Putting the private key option content in the error message seems like a bad idea.  Even if it's invalid, it may be close enough to valid and the error gets logged or exposed insecurely and leaks the key.

Reply:
Good point, only the header is printed out now. Fixed in 2db6cc7ae7988dde2660d438b538777442a1c67a.